src/inmanta/agent/forking_executor.py

```diff
@@ -224,6 +246,22 @@ async def call(self, context: ExecutorContext) -> typing.Sequence[inmanta.loader
                 logger.info("Failed to load sources: %s", module_source, exc_info=True)
                 failed.append(module_source)
 
+        if context.server.timer_venv_checkup is not None:
```

Comment:
init should not be called twice, so I don't know if this is required 

(we could make it fail if called twice, might be safer)